site/tsconfig.json

```diff
@@ -23,8 +23,8 @@
       "@components/*": ["components/*"],
       "@commerce": ["../packages/commerce/src"],
       "@commerce/*": ["../packages/commerce/src/*"],
-      "@framework": ["../packages/shopify/src"],
-      "@framework/*": ["../packages/shopify/src/*"]
+      "@framework": ["..\\packages\\shopify\\src"],
+      "@framework/*": ["..\\packages\\shopify\\src/*"]
```

Comment:
RIs this change necessary for the dependency upgrade? if not please revert it or make it in another PR